apps/dcos_dns/src/dcos_dns_mesos.erl

```diff
@@ -64,8 +64,9 @@ handle_info({timeout, _Ref, init}, #state{ref = undefined} = State) ->
     {noreply, State, {continue, init}};
 handle_info({timeout, Ref, masters}, #state{masters_ref = Ref} = State) ->
     {noreply, handle_masters(State)};
-handle_info({timeout, Ref, push_ops}, #state{ops_ref = Ref} = State) ->
-    {noreply, handle_push_ops(State), hibernate};
+handle_info({timeout, Ref, {push_zone, Rev}},
+            #state{push_zone_ref = Ref} = State) ->
```

Comment:
What happened to the indentation rules? :)

Reply:
Fixed, thanks a lot for catching. I like how you are committed to follow those insane rules :)

Reply:
Still trying to identify that person that contributes to `dcos-net` and uses `notepad.exe` instead of Emacs. Good thing it is not you :D